src/builder/BridgeRoutes.sol

```diff
@@ -196,4 +197,18 @@ library Across {
             )
         );
     }
+
+    // Returns whether or not an asset is bridged non-deterministically. This applies to WETH/ETH, where Across will send either ETH or WETH
+    // to the target address depending on if address is an EOA or contract.
+    function isNonDeterministicBridgeAction(HashMap.Map memory assetsBridged, string memory assetSymbol)
+        internal
+        pure
+        returns (bool)
+    {
+        uint256 bridgedAmount = HashMap.contains(assetsBridged, abi.encode(assetSymbol))
```

Comment:
Does the hashmap return zero for non-existent values or does it revert? Since otherwise this could just default to zero as a valid code path?

Reply:
It current reverts with `KeyNotFound`

Reply:
Minor facepalm, but okay. I guess lack of generics and therefore default value